version/version.go

```diff
@@ -8,7 +8,7 @@ package version
 // Version information that is displayed by the "version" command and used to
 // identify the version of running instances of OPA.
 var (
-	Version   = ""
+	Version   = "1.0.4"
```

Comment:
❓ What is this about? IIRC, the proper version is set at buildtime here. 🤔

Reply:
By accident, pushing a fix - sorry